lib/browser/mixins/browser_wait.ts

```diff
@@ -1,3 +1,4 @@
+import { errors as puppeteerErrors } from 'puppeteer';
```

Comment:
Try to wrap these errors inside [puppeteer_wrapper](https://github.com/angrykoala/wendigo/tree/master/lib/puppeteer_wrapper) feel free to add them to puppeteer_types or create a new file, the idea is that anything that is from puppeteer goes through an interface to simplify core changes when Puppeteer is updated or, eventually, use a different engine if needed